doc/design.adoc

```diff
@@ -94,13 +94,16 @@ is gracefully shut down.
 | v1
 
 | `expose`
-| Expose Infinispan for external access over port 11222.
-At the very least,
-the user must indicate via `type` attribute how the
+| If not empty the operator will expose Infinispan port 11222 externally.
+The `spec.type` attribute must indicate how the
 https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.11/#service-v1-core[Service]
-is exposed.
-`NodePort` is often used while testing locally,
-while `LoadBalancer` is used in production-like environments where an external load-balancer is available.
+is exposed. Two service types are supported: `LoadBalancer, NodePort`.
+For the `NodePort` service type the `Spec.Ports[0].nodePort` attribute can be specified, otherwise services
+are exposed with default value.
```

Comment:
`with default value` seems a bit misleading as a node port is selected from the free ports by OpenShift. (What is the default value?)